inputs-required.tf

```diff
@@ -64,7 +68,15 @@ variable "vm_availabilty_zones" {
 
 variable "network_interfaces" {
   description = "One or more network_interface can be set here, at least one of the network_interface have to be primary"
+  default     = ""
 }
+
+# variable "network_interface_ids" {
+#   description = "List of network interface IDs"
+#   type        = list(string)
+#   default     = []
+# }
```

Comment:
Is this variable not required?

Reply:
If it isn't needed, could we double check no one is using/referencing it. If it isn't being used then can it just be removed please?